test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: circt-opt -pass-pipeline='firrtl.circuit(firrtl-lower-types)' -split-input-file %s | FileCheck %s
+// RUN: circt-opt -pass-pipeline='firrtl.circuit(firrtl-lower-bundle-vectors)' %s | FileCheck %s
```

Comment:
Nit: Is the removal of `-split-input-file` a preference thing or is there some benefit to not splitting?

Reply:
I think things a bit quicker if it's not split.  But what I really like is the errors are relative to the original source, not offset from a split.